### DISCUSSION TITLE
Export to Logs button disabled

### DISCUSSION THREAD

**Discussion by nathanielks** [noop-inc/documentation#86] [Bugs] [answered]
I was trying to get an export of the logs for a previous bug report, but the button is disabled:

![Screenshot 2024-01-23 at 11 52 58 AM](https://github.com/noop-inc/documentation/assets/314009/5a89042b-9b8c-431d-b34f-5d2d9b905e5e)

**Comment by derekwolpert** [comment, marked as answer]
Thanks for the heads up here. A fix for this issue will be included in the next Noop Desktop release.
